src/compiler/commandLineParser.ts

```diff
@@ -816,6 +816,14 @@ const commandOptionsWithoutBuild: CommandLineOption[] = [
         description: Diagnostics.Do_not_transform_or_elide_any_imports_or_exports_not_marked_as_type_only_ensuring_they_are_written_in_the_output_file_s_format_based_on_the_module_setting,
         defaultValueDescription: false,
     },
+    {
+        name: "isolatedDeclarations",
+        type: "boolean",
+        category: Diagnostics.Interop_Constraints,
+        description: Diagnostics.Ensure_that_each_file_can_have_declaration_emit_generated_without_type_information,
+        transpileOptionValue: true,
+        defaultValueDescription: false,

```

Comment:
this should have affectsEmit and affectsBuildInfo

Reply:
Added . Thank you.